### ISSUE TITLE
Latest source code

### ISSUE THREAD

**Issue by artikz** [mvarnagiris/financius#406]
Hello!

As you are not developing this app anymore, could you please upload the latest code here?

**Comment by iamtodor** [comment]
@mvarnagiris could you please take a look at the topic?

**Comment by mvarnagiris** [comment]
Yes, sorry about that. I will try. Problematic thing is that at the moment it exposes private keys. I will need to sort that out

**Comment by andreskrey** [comment]
FYI you can use BFG to remove files from the repo and force push to make the repo "forget" about those credentials.

https://help.github.com/articles/removing-sensitive-data-from-a-repository/

**Comment by mvarnagiris** [comment]
OK I've pushed latest source code but you will need to help me to make it compile properly. https://github.com/mvarnagiris/financius/tree/latest

**Comment by FranciscodeMaussion** [comment]
something here?
